charts/kubescape-operator/README.md

```diff
@@ -1212,3 +1213,17 @@ To retrieve the `NetworkNeighbors` CRD for a workload, you can run the following
 ```bash
 kubectl -n <namespace> get networkneighbors <workload-kind>-<workload-name> -o yaml
 ```
+---
+## Common Issues
+* Error starting the container watcher - `(fanotify)`.
+
+  This error is usually caused by the `node-agent` not being able to find `runc` in any of the default paths.
+  This can be fixed by adding the path of `runc` to the global configuration [here](#values).
+  If you aren't sure where `runc` is located, you can run the following command on the node to find it:
+  ```bash
+  find / -name runc 2>/dev/null
+  ```
+  In case you are in an environment where you can't access the node, once solution is to run a privileged pod on the node, and run the command from there, to run a privileged pod, run the following command:
```

Comment:
```suggestion
  In case you are in an environment where you can't access the node, one solution is to run a privileged pod on the node, and run the command from there. To create a privileged pod, run the following command:
```